src/styles/Book.module.css

```diff
@@ -0,0 +1,200 @@
+@import url('https://fonts.googleapis.com/css2?family=Agdasima&family=Montserrat:wght@300;400;500;600&family=Roboto+Mono&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Agdasima&family=Montserrat:wght@300;400;500;600&family=Roboto+Mono&family=Roboto+Slab:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Agdasima&family=Montserrat:wght@300;400;500;600&family=Roboto+Mono&family=Roboto+Slab:wght@300;500&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --skyBlue: #0290ff;
+  --light: #fff;
+  --greyLight: rgb(222, 222, 210);
+  --greyDark: #69686c;
+  --dark: #000;
+  --whiteIn: #e8e8e8;
+  --greyDeep: #f5f6fa;
+  --blueFade: #4386bf;
+  --white-three: #e8e8e8;
+  --black-one: #121212;
+  --blue-one: #4386bf;
+}
+
+.li {
+  list-style: none;
+}
+
+.bookList {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.8rem;
+  padding: 0;
+}
+
+.lilLine {
+  width: 0.125rem;
+  height: 1.5rem;
+  border: solid 0.06rem var(--greyLight);
+  background-color: var(--greyLight);
+}
+
+.section {
+  display: flex;
+  justify-content: space-between;
+  padding: 2rem 0.7rem;
+  padding-right: 7rem;
+  background-color: var(--light);
+}
```

Comment:
- [ ] Please add the `align-items: centre` property to align your child's content on the x-axis properly, because your child contents are not aligned right now but once you add the property everything will then look fine, more professional and more beautiful.

## Requirement
![Screenshot from 2023-08-25 19-39-00](https://github.com/kambereBr/bookstore/assets/66476329/0b10d308-79f2-4892-8ba9-ef2ab2dd4849)

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-08-26 17-04-22](https://github.com/kambereBr/bookstore/assets/66476329/40acde1d-1faa-4dca-ac12-123bb08f59b2)

</td>
<td>

![Screenshot from 2023-08-26 17-03-50](https://github.com/kambereBr/bookstore/assets/66476329/ea68c0e2-081f-4ef6-b059-6ac1ba4631dc)


</td>
</tr>
</table>